### DISCUSSION TITLE
Skotin í Sarajevo - 2024/1/5

### DISCUSSION THREAD

**Discussion by giscus** [Biskupsreglan/biskupssidan#247] [vefsida-ummæli]
# Skotin í Sarajevo - 2024/1/5

Heimasíða Biskupsreglunnar

https://www.biskupsreglan.com/verkefni/Gavrilo-Princip

**Comment by TJ478** [comment]
Ertu að tala um Gavrilo Mierdo eða Gavrilo Princip?
Kannski er Gavrilo Princip pabbi Gavrilo Mierdo?
